FESS-GUI/windowsnames.h

```diff
@@ -2,6 +2,12 @@
 #define WINDOWSNAMES
 
 #define MAINWINDOW_TITLE "FlyCAM - Flywheel Control and Monitoring"
+#define MAINWINDOW_ERROR "FlyCAM - Error"
```

Comment:
I prefer #defines from my experience in embedded systems. The advantages in using them there don't carry over to this application as much but they do still have advantages. Memory usage is one example. All of QT functions I have used are pass by value. It we defined all the strings as globals we would roughly double the amount of memory used to store titles and error messages. Global variables are also dangerous. Window's global interace struct is a perfect example of this. 

Some names are stored in the .ui files so those wouldn't need to be save in a separate file but Pop-ups don't have .ui files and need to have their names specified with the setWindowTitle function. I moved all names into a header file for consistency. 

The perfect solution would be a class for logging, errors, and settings. Have each read the desired text from a file and just pass those objects around as needed. If we want to do that it would be great but until then converting everything from #defines in just more work for little gain.

PS. The preprocessor is awesome. You could write entire programs using nothing but preprocessor directives. :)